src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/VariantsSparkSource.java

```diff
@@ -25,14 +27,29 @@ public VariantsSparkSource(JavaSparkContext ctx) {
 
     /**
      * Loads variants in parallel using Hadoop-BAM for vcfs and bcfs.
+     *
      * @param vcf file to load variants from.
-     * @return JavaRDD<GATKVariant> of variants from all files.
+     * @return JavaRDD<GATKVariant> of variants from the variants file specified in vcf.
      */
     public JavaRDD<GATKVariant> getParallelVariants(final String vcf) {
         return getParallelVariantContexts(vcf).filter(vc -> vc.getCommonInfo() != null).map(vc -> VariantContextVariantAdapter.sparkVariantAdapter(vc));
     }
 
     /**
+     * Loads multiple variants in parallel using Hadoop-BAM for vcfs and bcfs.
+     *
+     * @param vcfs List of files to load variants from.
+     * @return JavaRDD<VariantContext> rdd containing the union of all variants from the variant
+     * files specified in vcfs.
+     */
+    public JavaRDD<GATKVariant> getParallelVariants(final List<String> vcfs) {
+        return vcfs.stream()
+                .map(vcf -> getParallelVariants(vcf))
```

Comment:
This is operating on the vcfs serially, one after another -- should we be using parallelism here? (via, eg., `parallelStream()` instead of `stream()`)